cxx-qt-lib/src/lib.rs

```diff
@@ -67,244 +67,50 @@ macro_rules! sep {
     };
 }
 
+macro_rules! include_cxx_file {
```

Comment:
It would help to have documentation for this macro regarding using `sep!` for path separators.

Reply:
This macro has been removed now.